### PR TITLE
styled radio input

### PR DESCRIPTION
**Pull request by bethshook**

### CODE REVIEW

File: src/components/Radio.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+// https://moderncss.dev/pure-css-custom-styled-radio-buttons/
+const StyledInput = styled.input`
+  appearance: none;
+  /* For iOS < 15 to remove gradient background */
+  background-color: ${colors.palette.white};
+  font: inherit;
+  color: ${colors.palette.pale};
+  width: 2rem;
+  height: 2rem;
+  border: 1px solid currentColor;
```
Comment (jivey) [inline src/components/Radio.tsx:22]: Ohh currentColor is cool, I haven't seen that before

**Review by jivey — COMMENTED** [review]
Do you know if using https://react-spectrum.adobe.com/react-aria/RadioGroup.html would give us some advantages over our own component? After using it for the Tabs component, I'm starting to look there first for new stuff because of the accessibility defaults.

**Comment by bethshook** [comment]
@jivey let me see if I can quickly integrate their Radio component. RadioGroup would be trickier for my current purposes as I'm mapping inputs dynamically from a json config and I'm not sure how much input-specific logic we want there but it's worth thinking about. probably a lot in the components lib that we could swap out for react-aria

**Comment by bethshook** [comment]
looks like you need the full RadioGroup for controlled inputs

**Review by jivey — APPROVED** [review]

**Pull request merged**